src/resources/js/Pages/Employee/Logs/Index.vue

```diff
@@ -139,39 +139,23 @@ export default {
       type: String,
       default: '',
     },
-    url: {
-      type: String,
-      default: null,
-    },
     paginator: {
       type: Object,
       default: null,
     },
   },
-  create() {
-    window.addEventListener("load", this.onWindowLoad);
-  },
   data() {
     return {
-      filterByLogType: '',
-      logTypes,
+      filterByLogType: this.search,
     };
   },
   computed: {
-    filteredLogs() {
-      if (!this.filterByLogType) return this.logs;
-      window.location.href = this.url + '?filterByLogType=' + this.filterByLogType;
-      return false;
+    filterLogs() {
+      Inertia.reload({ data: { filterByLogType: this.filterByLogType } });
     },
   },
   methods: {
     toTitleCase,
-    onWindowLoad() {
-      console.log("window load event");
-      if(document.getElementById("current_search").value) {
```

Comment:
Usually any time I see a `document....` call in React or Vue, that's a good indicator that there is a different way to go about it. Since both React and Vue rely on the virtual DOM, there is almost always a "right" way to accomplish this same thing.